fpc_imp_yoshino.c

```diff
@@ -294,6 +294,11 @@ int64_t fpc_load_db_id(fpc_imp_data_t *data)
     ALOGD(__func__);
     fpc_data_t *ldata = (fpc_data_t*)data;
 
+    // return cached auth_id value if available
+    if(ldata->auth_id) {
+        ALOGD("%s: Already have auth_id, returning %lu\n",__func__,ldata->auth_id);
```

Comment:
vendor/sony-oss/fingerprint/fpc_imp_loire_tone.c:304:68: warning: format specifies type 'unsigned long' but the argument has type 'uint32_t' (aka 'unsigned int') [-Wformat]
        ALOGD("%s: Already have auth_id, returning %lu\n",__func__,ldata->auth_id);